http_types/utils.py

```diff
@@ -179,6 +198,11 @@ def from_dict(obj: Any) -> Response:
             body_as_json = parse_body(obj_copy['body'])
             obj_copy['bodyAsJson'] = body_as_json
 
+        if "timestamp" in obj_copy:
+            obj_copy['timestamp'] = datetime.fromisoformat(obj_copy['timestamp'])
+        else:
+            obj_copy['timestamp'] = None
```

Comment:
I think you can remove this if `timestamp` is not present and it should work?

Reply:
Otherwise `typeguard.check_type` throws, and we would need to remove all calls to that.

Perhaps it's time to drop `TypedDict` for at least Request and Response and use normal classes?